docs/maintainer/knowledge_base.md

```diff
@@ -768,44 +768,45 @@ In the case of linking, you need to use the `pin_compatible` function to ensure
 ```yaml
 host:
   - numpy
-run:
-  - {{ pin_compatible('numpy') }}
 ```
 
-At the time of writing (January 22, 2022), above is equivalent to the following,
+At the time of writing (January, 2025), above is equivalent to the following,
 
 ```yaml
 host:
-  - numpy   1.18   # [py==37]
-  - numpy   1.18   # [py==38]
-  - numpy   1.19   # [py==39]
-run:
-  - numpy >=1.18.5,<2.0.a0   # [py==37]
-  - numpy >=1.18.5,<2.0.a0   # [py==38]
-  - numpy >=1.19.5,<2.0.a0   # [py==39]
+  - numpy   1.22   # [py==39]
+  - numpy   1.22   # [py==310]
+  - numpy   1.23   # [py==311]
+  - numpy   1.26   # [py==312]
+```
+though the ongoing migration for numpy 2.0 has already been applied to many
+feedstocks, in which case the pinning looks like
+```yaml
+host:
+  - numpy   2.0    # [py==39]
+  - numpy   2.0    # [py==310]
+  - numpy   2.0    # [py==311]
+  - numpy   2.0    # [py==312]
 ```
 
 See the pinning repository for what the pinning corresponds to at time of writing
-[https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L631](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L631)
+[https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L742](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L742)
 
-:::note[Notes]
-
-1. You still need to respect minimum supported version of `numpy` for the package!
-   That means you cannot use `numpy 1.9` if the project requires at least `numpy 1.12`,
-   adjust the minimum version accordingly!
+In either case, the actual runtime requirements are determined through numpy's-are-some-good-examples
```

Comment:
```suggestion
In either case, the actual runtime requirements are determined through numpy's
```

Reply:
huh, that must have been a stray editor suggestion, sorry